models/public/erfnet/accuracy-check.yml

```diff
@@ -0,0 +1,49 @@
+models:
+  - name: ERFNet
+    launchers:
+      - framework: onnx_runtime
+        adapter:
+          type: segmentation
+          make_argmax: True
+    datasets:
+      - name: erfnet_data
+        preprocessing:
+          - type: resize
+            dst_width: 976
+            dst_height: 208 
+        postprocessing:
+          - type: resize_segmentation_mask
+            dst_width: 1640
+            dst_height: 350
+            apply_to: prediction
+        metrics:
+          - type: segmentation_accuracy
+            use_argmax: false
+            presenter:  print_scalar
+          - type: mean_iou
+            use_argmax: false
+            presenter:  print_scalar
+  - name: ERFNet
+    launchers:
+      - framework: dlsdk
+        adapter:
+          type: segmentation
+          make_argmax: True
+    datasets:
+      - name: erfnet_data
+        preprocessing:
+          - type: resize
+            dst_width: 976
+            dst_height: 208
+        postprocessing:
+          - type: resize_segmentation_mask
+            dst_width: 1640
+            dst_height: 350
+            apply_to: prediction
+        metrics:
+          - type: segmentation_accuracy
+            use_argmax: false
+            presenter: print_scalar
```

Comment:
please provide metric reference to config

Reply:
sorry,I don't understand what configuration needs to be provided
Could you provide an example?thank you

Reply:
Or is it possible to delete segmentation_accuracy metics in accuracy-ckeck.yml, because I only wrote the test result of mean_iou in the README document.

Reply:
like this:
reference: 0.7647
l know how to slove it.
thank you